### ISSUE TITLE
Support additional neural network layers

### ISSUE THREAD

**Issue by supreme-gg-gg** [supreme-gg-gg/block-ml#7]
US:
- Support not only DNN but other CNN, RNN, Dropout, Flatten layers 
- Many are just copy and pasting and referencing Keras documentation
- No need to allow full customisation just select the key ones
- Pay attention to dimensional analysis to avoid bug with script
AC:
- New layers blocks created as JSON and translation rules set in pythonGenerator
- Blocks tested to integrate well with existing layers 
- you can still use the sequential layer wrapper even with other layers
- Script generated tested and executed on sample data
